### ISSUE TITLE
error parsing the glyph

### ISSUE THREAD

**Issue by omridevk** [nfroidure/svgicons2svgfont#162]
## Issue

```
Error: Got an error parsing the glyph "Download-circle_duotone_line": Unexpected character "N" at index 1. Command cannot follow comma.
```
and here's the svg in question:
![Download-circle_duotone](https://user-images.githubusercontent.com/9701896/185768478-0c908c68-67b2-4943-ae6c-b37a4db6c61f.svg)

I have tried to remove the transforms, but it still fails

I'm a gentledev I:
- [X] fully read the README recently
- [X] searched for existing issues
- [X] checked I'm up to date with the latest version of the project

### Debugging informations
- `node -v` result:
```
v16.15.1
```

- `npm -v` result:
```
8.11.0
```
If the result is lower than 16.15.0, there is
 poor chances I even have a look to it. Please,
 use the last [NodeJS LTS version](https://nodejs.org/en/).

**Comment by Carlos-Henreis** [comment]
Did you solve the problem? I have many cases like this...

**Comment by nfroidure** [comment]
@Carlos-Henreis the above issue will be fixed in the 14.0.2 version. If it doesn't fix your problems, pleasde open another issue and provide the problematic icons. Feel free to PR too according to the commit that closes this issue if you want to give a try for a bug fix.
